src/python/pull_gsheets_translations_mvp.py

```diff
@@ -0,0 +1,299 @@
+		#make fieldKey and add to new list of fieldKeys
+		if len(verbs)==0 and len(nouns)==2:
+			lst_new.append(nouns[0].lower() + nouns[1].title())
+		elif len(verbs)==1 and len(nouns)==1:
+			lst_new.append(nouns[0].lower() + verbs[0].title())
+		else:
+			lst_new.append(nouns[0].lower() + nouns[1].title() + verbs[0].title())
+	return Series(lst_new,index=language_df.index)
+
+def survey_value_cleaner(language_df):
+	""" Converts the long text found within the 'Survey' sheets to shorter text to be used as JSON keys.
+
+	Arguments:
+		language_df {[DataFrame]} -- [A pandas DataFrame containing the information from the 'Survey' sheet.]
+
+	Returns:
+		[DataFrame] -- [Returns a new DataFrame with shortended text and no punctuation.]
+	"""
+	vPA_col = [i for i,x in enumerate(language_df.columns) if x=='valuePossibleAnswers'][0]
+	tVPA_col = [i for i,x in enumerate(language_df.columns) if x=='translatedValuePossibleAnswers'][0]
+	v_col = [i for i,x in enumerate(language_df.columns) if x=='value'][0]
+	tV_col = [i for i,x in enumerate(language_df.columns) if x=='translatedValue'][0]
+	rows = language_df.values.tolist()
+	rows_added = []
+	for i,row in enumerate(rows):
+		orig_row = row.copy()
+		orig_row[v_col] = convert_to_camelCase(orig_row[v_col])
+		rows_added.append(orig_row)
+		arr = row[vPA_col].split(';')
+		tarr = row[tVPA_col].split(';')
+		if len(arr)==len(tarr) and len(arr)>3:
+			for j,_ in enumerate(arr):
+				row_new = row.copy()
+				row_new[v_col] = arr[j].replace('\n','')
+				row_new[v_col] = convert_to_camelCase(row_new[v_col]).replace(" ","")
+				#.replace(' ','').translate(str.maketrans('', '', punctuation))
+				row_new[tV_col] = tarr[j].replace('\n','')
+				#.replace(' ','').translate(str.maketrans('', '', punctuation))
+				rows_added.append(row_new)
+	return DataFrame(rows_added,columns=language_df.columns)
+
+def convert_worksheet_to_df(wk):
+	"""Converts current worksheet into a pandas dataframe.
+
+	Arguments:
+		wk {[worksheet object]} -- [Current language worksheet being iterated over.]
+
+	Returns:
+		[DataFrame] -- [A pandas dataframe containing the information from the current worksheet with a cleaned 'parentKey' column.]
+	"""
+	language_lists = wk.get_all_values()
+	language_df = DataFrame(language_lists[1:],columns=language_lists[0])
+	language_df = language_df[language_df['parentKey']!='']
+	return language_df
+
+def clean_dataframe_column_values_to_short_JSON_keys(wk,df):
+	""" Cleans the current language dataframe columns into short JSON keys, depending on the type of worksheet.
+
+	Arguments:
+		wk {[worksheet object]} -- [Current worksheet being iterated over.]
+		df {[DataFrame]} -- [The current language dataframe.]
+
+	Returns:
+		[DataFrame] -- [A cleaned dataframe with the necessary columns for the JSON key:value pairs.]
+	"""
+	if 'Education' in wk.title:
+		df.value = education_value_cleaner(df)
+	elif 'Survey' in wk.title:
+		osLanguage_df = df
+		df = survey_value_cleaner(df)

```

Comment:
It would be more consistent if `education_value_cleaner` and `survey_value_cleaner` returned the same type